features/dimensions.private.feature

```diff
@@ -128,3 +161,48 @@ Feature: Dimensionas
         ]
     }
     """
+
+ Scenario: Getting dimensions description
+    Given I am identified as "user@ons.gov.uk"
+
+    And I am authorised
+
+    And I have the following population types in cantabular
+    """
+    ["Example", "Example2"]
```

Comment:
When you update your branch you'll see this will need to change (now response contains Label and Name, not just single strings)